DevCycle.SDK.Server.Common/Model/Local/DVCLocalOptions.cs

```diff
@@ -1,17 +1,22 @@
 namespace DevCycle.SDK.Server.Common.Model.Local
 {
-    public class DVCLocalOptions: IDVCOptions
+    public class DVCLocalOptions : IDVCOptions
     {
-        public int ConfigPollingIntervalMs { get; private set; }
-        public int ConfigPollingTimeoutMs { get; private set; }
-        public string CdnUri { get; private set; }
+        public int ConfigPollingIntervalMs { get; }
+        public int ConfigPollingTimeoutMs { get; }
+        public string CdnUri { get; }
+        public bool DisableEvents { get; }
```

Comment:
this should mirror what we did on Node, separating "DisableAutomaticEvents" from "DisableCustomEvents"

Reply:
will do